### PR TITLE
Add HealthState field to Cluster type.

### PR DESCRIPTION
**Pull request by nimrodshn**
cc: @igoihman @vkareh

### CODE REVIEW

File: model/clusters_mgmt/v1/cluster_health_state_type.model [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// ClusterHealthState indicates the health of a cluster.
+enum ClusterHealthState {
+    // Cluster is Ready and healthy.
+    ClusterHealthStateHealthy
```
Comment (vkareh) [inline model/clusters_mgmt/v1/cluster_health_state_type.model:20]: Indentation looks off here /nitpick

File: model/clusters_mgmt/v1/cluster_health_state_type.model [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+enum ClusterHealthState {
+    // Cluster is Ready and healthy.
+    ClusterHealthStateHealthy   
+
+    // Cluster is Ready and unhealthy.
+	ClusterHealthStateUnhealthy 
+
+    // Cluster health is unkown.
+	ClusterHealthStateUnknown
+}
```
Comment (vkareh) [inline model/clusters_mgmt/v1/cluster_health_state_type.model:27]: I think these health states should just be the actual value, like
```
enum ClusterHealthState {
  Healthy
  Unhealth
  Unknown
}
```
The reason for that is that the metadata generator for the SDK will take the entry from this file and auto-generate the allowed values for state, which in this case will end up looking like this:
```
{
  "health_state": "cluster_health_state_healthy"
}
```

Comment (nimrodshn) [reply model/clusters_mgmt/v1/cluster_health_state_type.model:27]: Ah ok cool

Comment (nimrodshn) [reply model/clusters_mgmt/v1/cluster_health_state_type.model:27]: Done.

File: model/clusters_mgmt/v1/cluster_health_state_type.model [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// ClusterHealthState indicates the health of a cluster.
+enum ClusterHealthState {
+  // Cluster is Ready and healthy.
+  Healthy
```
Comment (vkareh) [inline model/clusters_mgmt/v1/cluster_health_state_type.model:20]: indentation still an issue :smile: 
I think you have multiple spaces here, where the others have a single tab

Comment (nimrodshn) [reply model/clusters_mgmt/v1/cluster_health_state_type.model:20]: maybe now?? :man_facepalming:

**Pull request merged**